cli/cache/http_cache.rs

```diff
@@ -42,7 +41,7 @@ fn base_url_to_filename(url: &Url) -> Option<PathBuf> {
     }
     "data" | "blob" => (),
     scheme => {
-      error!("Don't know how to create cache name for scheme: {}", scheme);
```

Comment:
Now this error is more likely to happen because it might load in some random files that use a different scheme (ex. "ext:" scheme in our case). This could be especially annoying in the repl. I downgraded this to debug because it didn't seem that important to surface at an "error" level.